portal/src/main/webapp/js/src/oncoprint/webgl/src/js/oncoprintruleset.js

```diff
@@ -472,11 +472,17 @@ var LinearInterpRuleSet = (function () {
 	this.value_key = params.value_key;
 	this.value_range = params.value_range;
 	this.log_scale = params.log_scale; // boolean
-	this.inferred_value_range;
+
+	this.rangeTypes = {
+		'SIGNED': 'SIGNED',
+		'UNSIGNED': 'UNSIGNED',
+		'NON_POSITIVE': 'NON_POSITIVE'
```

Comment:
Can you explain a bit more about the naming choice here? What is the difference between SIGNED and NON_POSITIVE (i.e. negative)? Don't the negative values have a `-` sign? Or does SIGNED mean "mixed" (i.e. mix of positive and negative values)?